oioioi/base/registration_backend.py

```diff
@@ -63,18 +63,18 @@ def register(self, form):
 
 
 urlpatterns = [
-    re_path(r'^register/$', RegistrationView.as_view(), name='registration_register'),
+    re_path(r'^sign-up/$', RegistrationView.as_view(), name='sign-up'),
```

Comment:
I am not sure it is bad, just thinking if `sign-up` is unique enough

Reply:
I think that `create-account` will be better and sufficient. Also, then the Polish translation would be different from "rejestracja".

Reply:
I would like to say I only disliked url's name, not the url itself. As for the name I like *create account*, as it contains a noun making it more specific. In terms of translations which is the main point of this PR I consider "załóż konto" as good translation of "sign-up". Anything that won't be the same two words for making an account and enrolling to the contest will fix it for me.